### PR TITLE
Mpes updates to conversion input files

### PR DESCRIPTION
**Pull request by Arora0**

### CODE REVIEW

File: tests/data/dataconverter/readers/mpes/config_file.json [hunk 1]
```diff
@@ -176,7 +176,7 @@
   "/ENTRY[entry]/INSTRUMENT[instrument]/BEAM[beam_pump]/extent/@units": "µm",
   "/ENTRY[entry]/INSTRUMENT[instrument]/BEAM[beam_pump]/fluence": "@attrs:metadata/instrument/beam/pump/fluence",
   "/ENTRY[entry]/INSTRUMENT[instrument]/BEAM[beam_pump]/fluence/@units": "mJ/cm^2",
-  "/ENTRY[entry]/INSTRUMENT[instrument]/MANIPULATOR[manipulator]/sample_temperature": "@attrs:metadata/sample/temperature",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MANIPULATOR[manipulator]/sample_temperature": "@attrs:metadata/file/trARPES:Carving:TEMP_RBV",
```
Comment (domna) [inline tests/data/dataconverter/readers/mpes/config_file.json:179]: Do this path changes also work for the "older" files we use for the example (WSe, MoTe)?

**Review by domna — APPROVED** [review]
Looks good to me as long as the path changes do not break any of the old examples.
I can also update the regression log file if you like.

File: tests/data/dataconverter/readers/mpes/config_file.json (hunk 1 above)
Comment (Arora0) [reply tests/data/dataconverter/readers/mpes/config_file.json:179]: Yes, it is compatible with both the files.

File: tests/data/dataconverter/readers/mpes/eln_data.yaml [hunk 2]
```diff
@@ -1,9 +1,25 @@
+title: Valence Band Dynamics - 1030 nm linear p-polarized pump, 0.6 mJ/cm2 absorbed fluence
 Instrument:
   Analyzer:
-    energy_resolution: 110
+    energy_resolution: 110.0
     momentum_resolution: 0.08
     slow_axes: delay
-    spatial_resolution: 10
+    spatial_resolution: 10.0
```
[the remark this answers is not in the document]
Comment (Arora0) [reply tests/data/dataconverter/readers/mpes/eln_data.yaml:7]: Yes, these changes are to be used in the Nomad examples in the north repo. The same is reflected in this PR [https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-remote-tools-hub/-/merge_requests/98]

[the remark this answers is not in the document]
Comment (domna) [reply tests/data/dataconverter/readers/mpes/eln_data.yaml:7]: Ok, perfect. Then in the other repo also the schema files for the nomad eln need to be updated. I'll just do that so you don't have to bother with the nomad eln syntax.

**Pull request merged**